src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -348,12 +348,20 @@ protected function describeContainerDefinition(Definition $definition, array $op
                     $argumentsInformation[] = sprintf('Service(%s)', (string) $argument);
                 } elseif ($argument instanceof IteratorArgument) {
                     $argumentsInformation[] = sprintf('Iterator (%d element(s))', \count($argument->getValues()));
+                    foreach (array_map(function (Reference $value) {return (string) $value; }, $argument->getValues()) as $service) {
+                        $argumentsInformation[] = sprintf('- %s', $service);
```

Comment:
since this is a reference, it should be displayed as `Service(%s)`
fixed in 7a53e8d3f9 too